modal/queue.py

```diff
@@ -22,6 +22,12 @@ class _Queue(_Object, type_prefix="qu"):
 
     The queue can contain any object serializable by `cloudpickle`, including Modal objects.
 
+    **Lifetime of a queue and its contents**
+
+    A `Queue`'s lifetime matches the lifetime of the app it's attached to, but the contents expire after 30 days.
```

Comment:
Except when it's a deployed self-contained queue, i.e. `Queue.persisted()` / `Queue.from_name(, create_if_missing=True)` 🤡

Reply:
So I'm not sure if we should actually talk about lifetimes here, since it's more of a general concept and each object type can have both ephemeral and persisted variants

Reply:
No strong opinion, but this is where we document lifetimes for dicts https://modal.com/docs/reference/modal.Dict#modaldictdict. Where should we discuss lifetime/expiry instead?

Reply:
Discussed IRL, will refactor/move docs later.